src/spf-explainer/i18n/en/templates/skeletons/no_search.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
i18n files don't have copyright headers.
We can do this for all i18n files in separate PR later.